test/utils_test.go

```diff
@@ -52,6 +54,10 @@ func (n *node) ToString() string {
 	return fmt.Sprintf("GRPC.Address: %s", n.GRPCServer.Address())
 }
 
+const (
+	NEED_VERIFICATION_ON_PREFIX = "__verify__"
```

Comment:
isn't it easier to just pass a flag to the `PullFromAssemblers `  func if we want to check for missing or not? 
instead of encoding it inside the transaction?

Reply:
actually why not just always check if there are any missing txs in all tests